src/platform/mhv/components/MhvAlertConfirmEmail/MhvAlertConfirmEmail.jsx

```diff
@@ -68,10 +81,16 @@ const MhvAlertConfirmEmail = ({ recordEvent = recordAlertLoadEvent }) => {
   return emailAddress ? (
     <>
       {confirmSuccess && (
-        <AlertSystemResponseConfirmSuccess recordEvent={recordEvent} />
+        <AlertSystemResponseConfirmSuccess
+          recordEvent={recordEvent}
+          tabIndex={-1}
```

Comment:
Adding a tabIndex to a custom component won't have an effect unless that component uses tabIndex as a prop. I believe prop-types will throw a warning or error in this case.